### ISSUE TITLE
Images not displaying alt text (v.23 delivery)

### ISSUE THREAD

**Issue by cjlim1092** [Simon-Initiative/oli-torus#3410]
**Describe the bug**
(On migrated course) When there are images on a page with an alt text they don't display alt text on mouse hover. The alt text is present on authoring. Alt text should display on hover. 

**To Reproduce**
[Example tokamak link
](https://tokamak.oli.cmu.edu/sections/intro_stats_0308_jlnuc/page/003oi_linear_relationships_1_of_8_tp9im)[Example legacy link](https://oli.cmu.edu/jcourse/workbook/activity/page?context=9a55f70e0a0001dc318ebe76cff59ad5)

**Comment by darrensiegel** [comment]
I don't believe that this is a bug.  In Torus, when alternate text is specified by the author, the system is rendering the `alt` attribute for that `<img>` tag in delivery.  The fact that legacy shows a tooltip is due to the fact that the theme also renders the alt text as the `title` attribute (which browsers interpret and render as a tooltip).

The intent of the alt text is to instruct the browser what to display when: 
1. The image doesn't load
2. A screen read is being used. 

Torus is meeting that intent. Legacy never should have been setting the alt text as the tooltip.
